js/iiif-iabookreader_strawberry.js

```diff
@@ -40,7 +40,7 @@
                         iiifmanifest: strawberrySettings['manifest'],
                         iiifdefaultsequence: null, //If null given will use the first sequence found.
                         maxWidth: 800,
-                        imagesBaseURL: 'https://cdn.jsdelivr.net/gh/internetarchive/bookreader@4.40.3/BookReader/images/',
+                        imagesBaseURL: strawberrySettings['iareaderimagesbaseurl'],
```

Comment:
This JS here needs to check if strawberrySettings['iareaderimagesbaseurl'] is defined  (typeof !== "undefined")
before using it directly. You (sadly) will have to hardcode also (as a fallback) the https://cdn.jsdelivr.net/gh/internetarchive/bookreader@4.40.3/BookReader/images/' string here. Why? Because after updating the code many other Archipelago users will have still their Display Modes Cached, but the JS will update immediately which means that new variable will not be passed at all until someone either clears caches (expensive on a large repo) or save the formatter settings manually